ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -794,10 +807,11 @@ static ProtocolSpecBuilder pragueDefinition(
         .precompileContractRegistryBuilder(MainnetPrecompiledContractRegistries::prague)
 
         // EIP-7002 Withdrawals / EIP-6610 Deposits / EIP-7685 Requests
-        .requestsValidator(pragueRequestsValidator(depositContractAddress))
+        .requestsValidator(pragueRequestsValidator(requestContractAddresses.getWithdrawalRequestContractAddress()))
+        .requestsValidator(pragueRequestsValidator(requestContractAddresses.getDepositContractAddress()))
+        .requestsValidator(pragueRequestsValidator(requestContractAddresses.getConsolidationRequestContractAddress()))
```

Comment:
This should be set only once and to the deposit address as `pragueRequestsValidator` expects only the Deposit address.

Reply:
`        .requestsValidator(pragueRequestsValidator(requestContractAddresses))`